# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"

"POT-Creation-Date: 2015-10-27 13:58+0100\n"

"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"


#: sdk/remote.py:139
msgid "Cannot store tokens in keychain, there might be an OS permission issue!"
msgstr ""

#: sdk/remote.py:158
msgid "Authentication Error"
msgstr ""

#: sdk/remote.py:209 sdk/remote.py:267
msgid "Unsupported HTTP method"
msgstr ""

#: sdk/remote.py:212 sdk/remote.py:270
msgid "Authentication Exception"
msgstr ""

#: sdk/remote.py:349
msgid ""
"Invalid JSON value received while getting remote changes. Is the server "
"correctly configured?"
msgstr ""

#: sdk/remote.py:386
msgid "Invalid JSON value received while getting remote changes"
msgstr ""

#: sdk/remote.py:413 sdk/remote.py:456 sdk/remote.py:749 sdk/remote.py:1013
msgid "Task interrupted by user"
msgstr ""

#: sdk/remote.py:652
msgid "Local file to upload not found!"
msgstr ""

#: sdk/remote.py:657
msgid "File not correct after upload (expected size was 0 bytes)"
msgstr ""

#: sdk/remote.py:696
msgid "File is incorrect after upload"
msgstr ""

#: sdk/remote.py:709
msgid "Original file was not found on server"
msgstr ""

#: sdk/remote.py:766
msgid "File not found after download"
msgstr ""

#: sdk/remote.py:771
msgid "File is not correct after download"
msgstr ""

#: sdk/remote.py:790

#, python-format
msgid "Error while downloading file: %s"
msgstr ""

#: sdk/exceptions.py:41
msgid "Http-Basic authentication failed, wrong credentials?"
msgstr ""

#: sdk/exceptions.py:45
msgid "Token-based authentication failed, reload credentials?"
msgstr ""

#: sdk/exceptions.py:49
msgid ""
"Token-based authentication seems unsupported, this may impact performances?"
msgstr ""

#: sdk/exceptions.py:60

#, python-format
msgid ""
"[Quota limit reached] - You are using %(usage)s iMB of %(total)s iMB, you "
"cannot upload %(filename)s %(filesize)s iMB"
msgstr ""

#: sdk/exceptions.py:65
#, python-format
msgid "[File permission] %s"
msgstr ""

#: sdk/exceptions.py:70
msgid "Stopping tasks"
msgstr ""

#: sdk/local.py:106
msgid "Cannot remove local folder"
msgstr ""

#: job/local_watcher.py:119
msgid "Walking through your local folder, please wait..."
msgstr ""

#: job/local_watcher.py:130
#, python-format
msgid "Detected %i local changes..."
msgstr ""

#: job/continous_merger.py:288
msgid "Job Paused"
msgstr ""

#: job/continous_merger.py:298
msgid "Job Started"
msgstr ""

#: job/continous_merger.py:308
msgid "Job stopping"
msgstr ""

#: job/continous_merger.py:334
msgid "Synchronized"
msgstr ""

#: job/continous_merger.py:358
msgid "Checking changes since last launch..."
msgstr ""

#: job/continous_merger.py:368
msgid "Oops, error while indexing the local folder. Pausing the task."
msgstr ""

#: job/continous_merger.py:390
msgid "Status: Paused"
msgstr ""

#: job/continous_merger.py:399
#, python-format
msgid "Status: scheduled for %s"
msgstr ""

#: job/continous_merger.py:406
#, python-format
msgid ""
"Cannot find local folder! Did you disconnect a volume? Waiting %s seconds "
"before retry"
msgstr ""

#: job/continous_merger.py:408
msgid "Cannot find local folder, did you disconnect a volume?"
msgstr ""

#: job/continous_merger.py:427
msgid "Gathering data from remote workspace, this can take a while..."
msgstr ""

#: job/continous_merger.py:436
#, python-format
msgid "No Internet connection detected! Waiting for %s seconds to retry"
msgstr ""

#: job/continous_merger.py:438
#, python-format
msgid ""
"Connection to server failed, server is probably down. Waiting %s seconds to "
"retry"
msgstr ""

#: job/continous_merger.py:447
#, python-format
msgid "Error while connecting to remote server (%s)"
msgstr ""

#: job/continous_merger.py:475
msgid "Merging changes between remote and local, please wait..."
msgstr ""

#: job/continous_merger.py:496
msgid "Conflicts detected, cannot continue!"
msgstr ""

#: job/continous_merger.py:514

#, python-format
msgid "Processing %i changes"
msgstr ""

#: job/continous_merger.py:553 job/continous_merger.py:555

#, python-format
msgid "%i files modified"
msgstr ""

#: job/continous_merger.py:564
msgid "An SSL error happened, please check the logs"
msgstr ""

#: job/continous_merger.py:567 ui/web_api.py:316
msgid "A proxy error happened, please check the logs"
msgstr ""

#: job/continous_merger.py:570 ui/web_api.py:319
msgid "Connection error: too many redirects"
msgstr ""

#: job/continous_merger.py:573 ui/web_api.py:322
msgid "Chunked encoding error, please check the logs"
msgstr ""

#: job/continous_merger.py:576 ui/web_api.py:325
msgid "Content Decoding error, please check the logs"
msgstr ""

#: job/continous_merger.py:579 ui/web_api.py:328
msgid "Http connection error: invalid schema."
msgstr ""

#: job/continous_merger.py:582 ui/web_api.py:331
msgid "Http connection error: invalid URL."
msgstr ""

#: job/continous_merger.py:585
msgid "Connection timeout, will retry later."
msgstr ""

#: job/continous_merger.py:588 ui/web_api.py:341
msgid "Cannot resolve domain!"
msgstr ""

#: job/continous_merger.py:592

#, python-format
msgid "Unexpected Error: %s"
msgstr ""

#: job/change_processor.py:155
#, python-format
msgid "New folder created at %s"
msgstr ""

#: job/change_processor.py:161 job/change_processor.py:168
#, python-format
msgid "Folder created at %s"
msgstr ""

#: job/change_processor.py:180
#, python-format
msgid "Deleted file %s"
msgstr ""

#: job/change_processor.py:186
#, python-format
msgid "Folder %s deleted"
msgstr ""

#: job/change_processor.py:197 job/change_processor.py:206
#, python-format
msgid "Moved %(source)s to %(target)s"
msgstr ""

#: job/change_processor.py:233
#, python-format
msgid "File %s downloaded from server"
msgstr ""

#: job/change_processor.py:266
#, python-format
msgid "File %s uploaded to server"
msgstr ""

#: job/change_processor.py:271
#, python-format
msgid "New file created at %s"
msgstr ""

#: job/change_processor.py:277
#, python-format
msgid "File created at %s"

msgstr ""

#: res/i18n/html_strings.py:2
msgid "How can I find my server URL?"
msgstr ""

#: res/i18n/html_strings.py:3
msgid ""
"The server URL is the adress that you can see in your browser when accessing "
"Pydio via the web."
msgstr ""

#: res/i18n/html_strings.py:4
msgid "It starts with http or https depending on your server configuration."
msgstr ""

#: res/i18n/html_strings.py:5
msgid ""
"If you are logged in Pydio and you see the last part of the URL starting "
"with \"ws-\", remove this part and only keep the beginning (see image below)."
msgstr ""

#: res/i18n/html_strings.py:6
msgid "Got it!"
msgstr ""

#: res/i18n/html_strings.py:7
msgid "Connecting ..."
msgstr ""

#: res/i18n/html_strings.py:8
msgid "Configure Connection"
msgstr ""

#: res/i18n/html_strings.py:9
msgid "Error while trying to connect to %1 :"
msgstr ""

#: res/i18n/html_strings.py:10
msgid "%1"
msgstr ""

#: res/i18n/html_strings.py:11
msgid ""
"Connect to the server with the same URL as the one you would use to access "
"through a web browser, e.g. http://mydomain.com/pydio"
msgstr ""

#: res/i18n/html_strings.py:12 res/i18n/html_strings.py:13

#: res/i18n/html_strings.py:14 res/i18n/html_strings.py:164

msgid "Required"
msgstr ""

#: res/i18n/html_strings.py:15
msgid "Tips"
msgstr ""

#: res/i18n/html_strings.py:16
msgid "where can I find the server URL?"
msgstr ""

#: res/i18n/html_strings.py:17
msgid "Connect"
msgstr ""

#: res/i18n/html_strings.py:18
msgid "Trust SSL certificate"
msgstr ""

#: res/i18n/html_strings.py:19
msgid ""
"Ooops, cannot contact agent! Make sure it is running correctly, process will "
"try to reconnect in 20s"
msgstr ""

#: res/i18n/html_strings.py:20

msgid "Select a workspace"
msgstr ""

#: res/i18n/html_strings.py:21
msgid "Full path to the local folder"
msgstr ""

#: res/i18n/html_strings.py:22
msgid "Are you sure you want to delete this synchro? No data will be deleted"
msgstr ""

#: res/i18n/html_strings.py:23
msgid "computing..."
msgstr ""

#: res/i18n/html_strings.py:24
msgid "SYNC %1 parameters"
msgstr ""

#: res/i18n/html_strings.py:25 res/i18n/html_strings.py:120
msgid "Server"
msgstr ""

#: res/i18n/html_strings.py:26 res/i18n/html_strings.py:35
#: res/i18n/html_strings.py:77 res/i18n/html_strings.py:121
msgid "Workspace"
msgstr ""

#: res/i18n/html_strings.py:27 res/i18n/html_strings.py:36
#: res/i18n/html_strings.py:78 res/i18n/html_strings.py:122
msgid "Folder"
msgstr ""

#: res/i18n/html_strings.py:28
msgid "Label"
msgstr ""

#: res/i18n/html_strings.py:29
msgid "Server Connexion"
msgstr ""

#: res/i18n/html_strings.py:30
msgid "Login"
msgstr ""

#: res/i18n/html_strings.py:31 res/i18n/html_strings.py:49
msgid "Password"
msgstr ""

#: res/i18n/html_strings.py:32
msgid "Local Folder"
msgstr ""

#: res/i18n/html_strings.py:33
msgid "Choose"
msgstr ""

#: res/i18n/html_strings.py:34 res/i18n/html_strings.py:72
msgid "Remote workspace"
msgstr ""

#: res/i18n/html_strings.py:37 res/i18n/html_strings.py:79
#: res/i18n/html_strings.py:118
msgid "Change"
msgstr ""

#: res/i18n/html_strings.py:38 res/i18n/html_strings.py:73
msgid "Synchronise only a subfolder of this workspace"
msgstr ""

#: res/i18n/html_strings.py:39 res/i18n/html_strings.py:75
#: res/i18n/html_strings.py:123
msgid "Whole workspace"
msgstr ""

#: res/i18n/html_strings.py:40
msgid "[loading...]"
msgstr ""

#: res/i18n/html_strings.py:41 res/i18n/html_strings.py:53
msgid "Advanced parameters"
msgstr ""

#: res/i18n/html_strings.py:42
msgid "Delete Sync"
msgstr ""

#: res/i18n/html_strings.py:43
msgid "Save Parameters"
msgstr ""

#: res/i18n/html_strings.py:44 res/i18n/html_strings.py:45
#: res/i18n/html_strings.py:81
msgid "Share %1 via Pydio"
msgstr ""

#: res/i18n/html_strings.py:46 res/i18n/html_strings.py:107
msgid "Description"
msgstr ""

#: res/i18n/html_strings.py:47
msgid "Path"
msgstr ""

#: res/i18n/html_strings.py:48
msgid "Security"
msgstr ""

#: res/i18n/html_strings.py:50
msgid "Expires after"
msgstr ""

#: res/i18n/html_strings.py:51
msgid "Days"
msgstr ""

#: res/i18n/html_strings.py:52
msgid "Downloads"
msgstr ""

#: res/i18n/html_strings.py:54
msgid "Generate Link"
msgstr ""

#: res/i18n/html_strings.py:55
msgid "SYNC NAME"
msgstr ""

#: res/i18n/html_strings.py:56
msgid "DATA SIZE"
msgstr ""

#: res/i18n/html_strings.py:57
msgid "ESTIMATED TIME"
msgstr ""

#: res/i18n/html_strings.py:58
msgid "Ready for ignition!"
msgstr ""

#: res/i18n/html_strings.py:59
msgid "Are you ready to launch the synchronization?"
msgstr ""

#: res/i18n/html_strings.py:60
msgid "Your data will be in orbit in no time!"
msgstr ""

#: res/i18n/html_strings.py:61
msgid "FIRE THE ROCKET!"
msgstr ""

#: res/i18n/html_strings.py:62
msgid "change parameters"
msgstr ""

#: res/i18n/html_strings.py:63
msgid "Synchronization running..."
msgstr ""

#: res/i18n/html_strings.py:64
msgid "Liftoff! First sync can take some time..."
msgstr ""

#: res/i18n/html_strings.py:65
msgid "CREATE NEW SYNCHRO"
msgstr ""

#: res/i18n/html_strings.py:66
msgid "DONE"
msgstr ""

#: res/i18n/html_strings.py:67
msgid "1/3 Select a workspace"
msgstr ""

#: res/i18n/html_strings.py:68
msgid "Welcome %1!"
msgstr ""

#: res/i18n/html_strings.py:69
msgid "You are connecting to %1"
msgstr ""

#: res/i18n/html_strings.py:70 res/i18n/html_strings.py:71
#: res/i18n/html_strings.py:124 res/i18n/html_strings.py:126
msgid "change"
msgstr ""

#: res/i18n/html_strings.py:74 res/i18n/html_strings.py:76
msgid "loading"
msgstr ""

#: res/i18n/html_strings.py:80 res/i18n/html_strings.py:130
#: res/i18n/html_strings.py:132
msgid "Next"
msgstr ""

#: res/i18n/html_strings.py:82
msgid "Shared Link"
msgstr ""

#: res/i18n/html_strings.py:83
msgid ""
"Shared link to the selected item already exists. Below is the link to the "
"selected item"
msgstr ""

#: res/i18n/html_strings.py:84
msgid "New shared link to the selected item has been generated"
msgstr ""

#: res/i18n/html_strings.py:85
msgid "Copy to Clipboard"
msgstr ""

#: res/i18n/html_strings.py:86
msgid "UnShare Link"
msgstr ""

#: res/i18n/html_strings.py:87
msgid "Sync Direction"
msgstr ""

#: res/i18n/html_strings.py:88
msgid ""
"By default, modifications from both sides (local/remote) are automatically "
"reflected on the other side."
msgstr ""

#: res/i18n/html_strings.py:89
msgid "Upload Only"
msgstr ""

#: res/i18n/html_strings.py:90
msgid "computer to server"
msgstr ""

#: res/i18n/html_strings.py:91
msgid "Bi-directional"
msgstr ""

#: res/i18n/html_strings.py:92
msgid "Download Only"
msgstr ""

#: res/i18n/html_strings.py:93
msgid "server to computer"
msgstr ""

#: res/i18n/html_strings.py:94
msgid "Sync Frequency"
msgstr ""

#: res/i18n/html_strings.py:95
msgid "By default, the application will synchronize automatically"
msgstr ""

#: res/i18n/html_strings.py:96
msgid "Automatically"
msgstr ""

#: res/i18n/html_strings.py:97
msgid "Manually"
msgstr ""

#: res/i18n/html_strings.py:98
msgid "Given Time"
msgstr ""

#: res/i18n/html_strings.py:99
msgid "Run the sync every day at"
msgstr ""

#: res/i18n/html_strings.py:100
msgid "Versionning"

msgstr ""

#: res/i18n/html_strings.py:107
msgid ""
"When files were modified on both the server and your computer at the same "
"time, synchronization is stopped."
msgstr ""

#: res/i18n/html_strings.py:102
msgid "Solve conflicts manually"
msgstr ""

#: res/i18n/html_strings.py:103
msgid "Always prefer local version"
msgstr ""

#: res/i18n/html_strings.py:104
msgid "Always prefer remote version"
msgstr ""

#: res/i18n/html_strings.py:105
msgid "Link Handle"
msgstr ""

#: res/i18n/html_strings.py:106
msgid ""
"If you want a durable and pretty link (like https://.../my-share-link), you "
"can use this field. Link handle will be generated if left empty."
msgstr ""

#: res/i18n/html_strings.py:108
msgid "This will be displayed to the shared users."
msgstr ""

#: res/i18n/html_strings.py:109
msgid "Access Rights"
msgstr ""

#: res/i18n/html_strings.py:110
msgid "By default, the item will be previewed and downloadable"
msgstr ""

#: res/i18n/html_strings.py:111
msgid "Preview"
msgstr ""

#: res/i18n/html_strings.py:112
msgid "Download"
msgstr ""

#: res/i18n/html_strings.py:113
msgid "Upload"
msgstr ""

#: res/i18n/html_strings.py:114 res/i18n/html_strings.py:128
msgid "Advanced Parameters"
msgstr ""

#: res/i18n/html_strings.py:115
msgid "Save changes"
msgstr ""

#: res/i18n/html_strings.py:116
msgid "2/3 Select a destination"
msgstr ""

#: res/i18n/html_strings.py:117
msgid "By default a local folder will be created on your computer"
msgstr ""

#: res/i18n/html_strings.py:119
msgid "3/3 Optional Parameters"
msgstr ""

#: res/i18n/html_strings.py:125
msgid "Local folder"
msgstr ""

#: res/i18n/html_strings.py:127
msgid "Name this synchro"
msgstr ""

#: res/i18n/html_strings.py:129 res/i18n/html_strings.py:131
msgid "Previous"
msgstr ""

#: res/i18n/html_strings.py:133 res/i18n/html_strings.py:137
#: res/i18n/html_strings.py:139 res/i18n/html_strings.py:143
msgid "Status"
msgstr ""

#: res/i18n/html_strings.py:134 res/i18n/html_strings.py:138
#: res/i18n/html_strings.py:140
msgid "syncing"
msgstr ""

#: res/i18n/html_strings.py:135
msgid "Size"
msgstr ""

#: res/i18n/html_strings.py:136
msgid "Estimated time"
msgstr ""

#: res/i18n/html_strings.py:141
msgid "Last sync"
msgstr ""

#: res/i18n/html_strings.py:142
msgid "ERROR"
msgstr ""

#: res/i18n/html_strings.py:144
msgid "idle"
msgstr ""

#: res/i18n/html_strings.py:145
msgid "[DISABLED]"
msgstr ""

#: res/i18n/html_strings.py:146
msgid "Conflicts"

msgstr ""

#: res/i18n/html_strings.py:147
msgid "Solve Conflict"
msgstr ""

#: res/i18n/html_strings.py:148
msgid "Solved"
msgstr ""

#: res/i18n/html_strings.py:149
msgid "Last files synced"
msgstr ""

#: res/i18n/html_strings.py:150
msgid "Open File"
msgstr ""

#: res/i18n/html_strings.py:151
msgid "Transfers in progress"
msgstr ""

#: res/i18n/html_strings.py:152
msgid ""
"An element named %1 was modified on both the server and on your local "
"computer. Select how you want to solve this conflicting case:"
msgstr ""

#: res/i18n/html_strings.py:153
msgid "Apply to all conflicts"
msgstr ""

#: res/i18n/html_strings.py:154
msgid "Mine"
msgstr ""

#: res/i18n/html_strings.py:155
msgid "Both Versions"
msgstr ""

#: res/i18n/html_strings.py:156
msgid "Theirs"
msgstr ""

#: res/i18n/html_strings.py:157 res/i18n/html_strings.py:158
msgid "Create a new synchronization"
msgstr ""

#: res/i18n/html_strings.py:159
msgid "Please wait ..."
msgstr ""

#: res/i18n/html_strings.py:160
msgid "Welcome to the Pydio Sync"
msgstr ""

#: res/i18n/html_strings.py:161
msgid "The easiest way to keep your data in control"
msgstr ""

#: res/i18n/html_strings.py:162
msgid "Loading..."
msgstr ""

#: res/i18n/html_strings.py:163 res/i18n/html_strings.py:165
msgid "Get Started"
msgstr ""

#: ui/web_api.py:300
msgid "Couldn't load your workspaces, check your server !"
msgstr ""

#: ui/web_api.py:302
msgid "Server not found (404), is it up and has it Pydio installed ?"
msgstr ""

#: ui/web_api.py:304
msgid "Authentication failed: please verify your login and password"
msgstr ""

#: ui/web_api.py:306
msgid "Access to the server is forbidden"
msgstr ""

#: ui/web_api.py:308
msgid "Server seems to be encountering problems (500)"
msgstr ""

#: ui/web_api.py:313

msgid ""
"An SSL error happened! Is your server using a self-signed certificate? In "
"that case please check 'Trust SSL certificate'"
msgstr ""

#: ui/web_api.py:338

msgid "Connection timeout!"
msgstr ""
